### PR TITLE
修复-底部滑动至页面顶部按钮被覆盖问题

### PR DESCRIPTION
**Pull request by gottayan**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: site/theme/static/footer.less [hunk 1]
```diff
@@ -7,7 +7,7 @@ footer {
   font-size: 14px;
   background-color: #000;
   position: relative;
-  z-index: 100;
```
Comment (tangjinzhou) [inline site/theme/static/footer.less]: 左下角有广告，不建议修改footer z-index
或许改变按钮的z-index更加合适

Comment (gottayan) [reply site/theme/static/footer.less]: 我没看到左下角有广告。
我看到右下角有广告。而且广告的z-index是9 按钮z-index是10 所以才考虑将footerz-index改为8.

Comment (tangjinzhou) [reply site/theme/static/footer.less]: 说错了  是右下角
拉到下方时，广告不要遮挡 footer   体验会更好些

Comment (gottayan) [reply site/theme/static/footer.less]: 哦好的

**Pull request closed without merging**